clustershell.spec.in

```diff
@@ -107,7 +111,9 @@ for i in clubak cluset clush nodeset; do
 done
 popd
 
+%if 0%{?rhel} < 8
 %{__python2} setup.py install -O1 --skip-build --root %{buildroot}
+%endif
```

Comment:
With this, you end up having versioned commands right, like clush-3.7, clubak-3.7, etc?
You should probably move the `%if 0%{?rhel} < 8` above, just before the pushd, so that commands in the python3-package won't be versioned at all. This is also what is recommended for the latest Fedora. Once your patch has landed, I will submit another patch for Fedora. Thanks!

Reply:
You are right, I forgot this point.

```
[root@management1 ~]# rpm -ql python3-clustershell-1.8.1-1.el8.noarch
/usr/bin/clubak-3.6
/usr/bin/cluset-3.6
/usr/bin/clush-3.6
/usr/bin/nodeset-3.6
/usr/lib/python3.6/site-packages/ClusterShell
```

Modified, and now I have the correct output:

```
[root@management1 ~]# rpm -ql python3-clustershell-1.8.1-1.el8.noarch
/usr/bin/clubak
/usr/bin/cluset
/usr/bin/clush
/usr/bin/nodeset
/usr/lib/python3.6/site-packages/ClusterShell
```

Thanks a lot for this clustershell/nodeset tools.
I and @oxedions are working actively on an opensource cluster stack project, and we are using them heavily.